io.github.ripose.Memento.yml

```diff
@@ -0,0 +1,577 @@
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: git
+        url: https://github.com/mpv-player/mpv.git
+        tag: v0.35.1
+        commit: 140ec21c89d671d392877a7f3b91d67e7d7b9239
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: libzip
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://libzip.org/download/libzip-1.9.2.tar.xz
+        sha256: c93e9852b7b2dc931197831438fee5295976ee0ba24f8524a8907be5c2ba5937
+        x-checker-data:
+          type: anitya
+          project-id: 10649
+          stable-only: true
+          url-template: https://libzip.org/download/libzip-$version.tar.xz
+
+  - name: json-c
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: git
+        url: https://github.com/json-c/json-c.git
+        tag: json-c-0.16-20220414
+        commit: 2f2ddc1f2dbca56c874e8f9c31b5b963202d80e7
+
+  - name: mecab
+    buildsystem: simple
+    cleanup:
+      - /bin
+      - /include
+      - /lib/mecab/mecab-*
+    build-commands:
+      - sed -i 's!prefix@/libexec!libexecdir@!g' mecab-config.in
+      - ./configure --prefix=/app --sysconfdir=/app/etc --libexecdir=/app/lib --with-charset=utf-8
+      - make
+      - make DESTDIR=/ install
+    sources:
+      - type: archive
+        archive-type: tar
+        url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/mecab/mecab-0.996.tar.gz
+        sha256: e073325783135b72e666145c781bb48fada583d5224fb2490fb6c1403ba69c59
+
+  - name: mecab-ipadic
+    buildsystem: simple
+    cleanup:
+      - /bin
+      - /include
+      - /lib/mecab/mecab-*
+    build-commands:
+      - ./configure --prefix=/ --sysconfdir=/etc --libexecdir=/lib --with-charset=utf-8 --with-mecab-config=/app/bin/mecab-config
+      - sed -i 's!libexec/mecab!lib/mecab!g' config.status
+      - make
+      - make DESTDIR=/ install
+    sources:
+      - type: archive
+        archive-type: tar
+        url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/mecab/mecab-ipadic-2.7.0-20070801.tar.gz
+        sha256: b62f527d881c504576baed9c6ef6561554658b175ce6ae0096a60307e49e3523
+
+  - name: memento
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DRELEASE_BUILD=ON
+    sources:
+      - type: git
+        url: https://github.com/ripose-jp/Memento.git
+        tag: v1.0.0
+        commit: 2a6eeaa97c18d24fe625cd8552effd43051ac5ac
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+      - type: file
+        path: io.github.ripose.Memento.appdata.xml
+      - type: shell
+        commands:
+          - mkdir -p /app/share/appdata
```

Comment:
these belong to `post-install`

Reply:
and the 3 command should be just using `install` (one-liner). In `post-install`

Reply:
Done